ColoradoSchoolOfMines/acm-discord-bot
# Safer Environmental Variables

Features
- Can optionally force returned env to be non-empty
- Loading .env moved to a function call
- If .env is not found, create a template in static/.env based on the ENVs enum

Tests
- Make sure that .env is the right shape based on the ENVs enum
- Make sure that .env is created successfully

Additionally, #1 will pass it's status checks once this branch is merged. Main error seems to be that it has no tests to run.
